incubator/hnc/hack/krew-kubectl-hns.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
```

Comment:
In the 0.6 PR you use sed to replace the tag, checksum, etc in this file. I don't see and of that here, am I missing something?

Reply:
oh I see, you meant to do it by hand. Maybe add a comment indicating as such? Or purposely making it invalid yaml so it isn't pushed accidentally?

Reply:
Big scary comment added :)